test-app/helm_deploy/hmpps-template-kotlin/values.yaml

```diff
@@ -30,6 +30,22 @@ generic-service:
   namespace_secrets:
     hmpps-template-kotlin:
       APPINSIGHTS_INSTRUMENTATIONKEY: "APPINSIGHTS_INSTRUMENTATIONKEY"
+    main-queue-sqs-instance-output:
+      HMPPS_SQS_QUEUES_MAINQUEUE_ACCESS_KEY_ID: "access_key_id"
+      HMPPS_SQS_QUEUES_MAINQUEUE_SECRET_ACCESS_KEY: "secret_access_key"
+      HMPPS_SQS_QUEUES_MAINQUEUE_QUEUE_NAME: "mainQueue"
+    main-queue-sqs-dl-instance-output:
+      HMPPS_SQS_QUEUES_MAINQUEUE_DLQ_ACCESS_KEY_ID: "dlq_access_key_id"
+      HMPPS_SQS_QUEUES_MAINQUEUE_DLQ_SECRET_ACCESS_KEY: "dlq_secret_access_key"
+      HMPPS_SQS_QUEUES_MAINQUEUE_DLQ_NAME: "mainDlq"
+    another-queue-sqs-instance-output:
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_ACCESS_KEY_ID: "another_access_key_id"
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_SECRET_ACCESS_KEY: "another_secret_access_key"
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_QUEUE_NAME: "anotherQueue"
+    another-queue-sqs-dl-instance-output:
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_DLQ_ACCESS_KEY_ID: "dlq_another_access_key_id"
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_DLQ_SECRET_ACCESS_KEY: "dlq_another_secret_access_key"
+      HMPPS_SQS_QUEUES_ANOTHERQUEUE_DLQ_NAME: "anotherDlq"
```

Comment:
Add examples of how to wire up Kube secrets into HmppsQueueProperties

Reply:
Oh and tried to write a test to prove that this format of env vars is wored into the HmppsQueueProperties but couldn't manage it - but did manage to test from the command line.